### ISSUE TITLE
Secondary degauss is included in the VESA.xml config, but this is a model specific function for 0x2 address

### ISSUE THREAD

**Issue by aqxa1** [ddccontrol/ddccontrol-db#123]
I've been spending some time building support for the LG 27GL850-B, and found that the secondary degauss function was added by this commit, as part of support for LG monitors in 2009:

>commit a132e1ec8859606fcdadb29dc2d2c62054def97c
Author: William Ollivier <guneemwelloeux@users.sourceforge.net>
Date:   Sun Nov 15 20:03:36 2009 +0000

    Few changes for the LG monitors
    Added LG F-Engine commands and support

This is linked to 0x2 address, however the MCCS standard suggests that 0x2 is for New Control Value (used for sychronization of display hardware and software, according to pg 102 of the [MCCS specification](https://milek7.pl/ddcbacklight/mccs.pdf)), and ddcutil reports the same:
>VCP code 0x02 (New control value             ): No new control values (0x01)

Possibly it should be kept as an option for older LG CRT monitors, but it causes problems for any modern monitors so should be moved out of the VESA.xml config.

**Comment by larstobi** [comment]
It is still possible to remove the option for a specific monitor. See #56 for the added documentation. I do see your point since basically no new CRT monitors are produced.

I am unsure how to remove it from the defaults without breaking functionality for old CRT monitors, though.

**Comment by SimonAlling** [comment]
> It is still possible to remove the option for a specific monitor. See #56 for the added documentation.

I believe you mean #127.

> I do see your point since basically no new CRT monitors are produced.
> 
> I am unsure how to remove it from the defaults without breaking functionality for old CRT monitors, though.

Are people actually using DDC Control for CRT monitors, and would removing the `secdegauss` control from the VESA profile be an actual problem for them? (I'm no CRT connoisseur by any stretch.)

**Comment by aqxa1** [comment]
My feeling is that it was never part of the standard at all, and was mistakenly included by the commit that added it in the first place (presumably, that monitor did have `secdegauss` at that address, but it's probably model specific). There's only three monitors that include reference to `secdegauss`, and the two other monitors that included it, are newer LCD monitors (so they aren't actually referencing `secdegauss` at that address, but some other function).

These look to be the other two monitors that have included the secdegauss option (both being LCD), assuming my search has turned up the correct monitors:
https://www.amazon.com/Dell-LED-Lit-Monitor-U2518D-Compatibility/dp/B075KGLYRL
https://www.acer.com/ac/en/US/content/model/UM.HE3AA.P01

But good to know that options can be disabled conditionally, I'll look into that.

**Comment by SimonAlling** [comment]
@aqxa1: Yup! Seems like the `secdegauss` control is merely a bug that recently turned ten years old. Should definitely be fixed.

**Comment by dunnmj42** [comment]
@aqxa1 I have a question for you specifically related to a LG 27GL83A-B (which both of my systems recognize as the 27GL850.

I am able to get both ddccontrol and the windows program "ControlMyMonitor" to change the input on this monitor from macOS and Win10 respectively, but in both cases, the changed input isn't reflected in the OSD. As an example, if I use ddccontrol to change the input from HDMI 1 (Mac) to Displayport (Win10) the monitors OSD shows it as HDMI 1 still being the input. The same thing happens in the other direction. (It might be worth noting that the first time after power up that I make this exact change from HDMI 1 to Displayport I get an "Out of Range" message on the monitor even though it displays correctly and the only way to get rid of that message is to use the OSD to change the input to match.)

I wonder if you have experienced a similar issue, and if so, if you have any idea why that might be happening.

It's also worth noting that both pieces of software are able to effectively and completely change the input on my other two monitors, both of which are ASUS monitors.

Alternatively, @larstobi or @SimonAlling if you have any insight; or would like me to start a new issue, I can do that.

**Comment by aqxa1** [comment]
@dunnmj42 Yeah, input switching is broken for me as well. The USB program on Windows does work for input switching, so it might be possible to reverse engineer the protocol and write something that works on Linux, but that is obviously a lot more work.

**Comment by SimonAlling** [comment]
@dunnmj42 and @aqxa1: Unfortunately I have no idea what might be causing that behavior. Is there any reason to suspect that it might be related to this issue? Otherwise, I think it's better to open a new one. I'd also say this issue was fixed in #128 and should be closed.

**Comment by SimonAlling** [comment]
Just a friendly reminder that this issue was fixed in #128 and should be closed. :slightly_smiling_face:
